app/renderer/abr-document.js

```diff
@@ -257,6 +262,10 @@ AbrDocument.prototype = {
         this.latestGeneration = this.getGeneration();
     },
 
+    setDirty: function () {
+        this.latestGeneration--;
```

Comment:
Good point.

The intention with `-1` would also be clearer; decrementing `latestGeneration` looks like we are stepping back in the undo/redo stack when it's not the case.